tests/unit/composables/Project.spec.js

```diff
@@ -237,9 +237,11 @@ describe('Test composable: Project', () => {
   });
 
   describe('Test function: initProject', () => {
-    it('should call git init', async () => {
-      const result = await initProject({ id: 'foo' });
-      expect(result).toEqual('init');
+    it('should init create commit with default file', async () => {
```

Comment:
` init create commit` -> `init and create commit` ?

Reply:
done